File: QTD2.Infrastructure/Model/RR_ILA_Link/RR_ILA_LinkOptions.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.RR_ILA_Link
{
    public class RR_ILA_LinkOptions
    {
        public int RegRequirementId { get; set; }

        public int[] IlaIds { get; set; }

        public string ChangeNotes { get; set; }

        public DateTime EffectiveDate { get; set; }
    }
}

File: QTD.SPA/src/app/_DtoModels/RR_ILA/RR_ILA_LinkOptions.ts
import { EnablingObjective } from "../EnablingObjective/EnablingObjective";
import { ILA } from "../ILA/ILA";
import { RegulatoryRequirement } from "../RegulatoryRequirements/RegulatoryRequirement";

export class RR_ILA_LinkOptions{
  regRequirementId!:any;
  ilaIds!:any[];
  changeNotes!:string;
  effectiveDate!:Date;
}
